plugin/trino-delta-lake/src/test/resources/deltalake/v2_checkpoint_parquet_using_temporal/README.md

```diff
@@ -0,0 +1,19 @@
+Data generated using Delta Lake 3.3.0:
+
+```sql
+SET spark.databricks.delta.checkpointV2.topLevelFileFormat = parquet;
+
+CREATE TABLE test_v2_checkpoint_parquet_using_temporal
```

Comment:
Is this table used in any of the tests?

Reply:
In the `TestDeltaLakeBasic#testTimeTravelWithV2CheckpointUsingTemporal`